src/charactersheet/components/index.js

```diff
@@ -10,3 +10,6 @@ import { NestedListComponentViewModel } from './nested-list';
 import { PlusMinusComponentViewModel } from './plus-minus';
 import { PreviewEditViewModel } from './preview-edit';
 import { ProficiencyTypeComponentViewModel } from './proficiency-marker';
+import './column-navigation';
```

Comment:
Why don't we have to name these imports like the other ones?

Reply:
We're not using it, just importing the binding. Technically we probably don't need to explicitly import any of them.

Reply:
nice. good to look out for in the future then